src/components/MenuBase/MenuBase.tsx

```diff
@@ -3,15 +3,38 @@ import { createContext, ReactElement } from "react";
 import MUIMenu from "@mui/material/Menu";
 
 import { MenuBaseProps, MenuBaseContextProps } from "./MenuBaseTypes";
+import styled from "@emotion/styled";
+import theme from "@root/theme";
 
 export const MenuBaseContext = createContext<MenuBaseContextProps>(null);
 
+const StyledMUIMenu = styled(MUIMenu)`
+	& > div {
+		box-shadow: 0px 2px 6px rgba(26, 26, 26, 0.16);
+		margin-top: 4px;
+
+		& > ul {

```

Comment:
If we observe this style we can see that it's actually not doing anything on the ul, this is a bad practice as it will make the component calculate a style that's just empty: 
![image](https://user-images.githubusercontent.com/23409284/193946312-cbbb6593-2e0d-408a-92d1-c5509856919d.png)

For future components what you could do is simply using regular combinators / selectors: https://www.w3schools.com/css/css_combinators.asp

E.g. & ul .MuiTouchRipple-child {} This way the style will only be added to the MuiTouchRipple element.

Reply:
Here we're assuming that MenuBase will always receive as children a ul, but in reality MenuBase can receive any ReactNode / JSX Element, so this css won't work for all cases, also it changes the style for whatever child element developers might pass. It's always a good practice to have each component deal internally with their own things. In this case, we know we want those styles to apply to the of MenuBase, so they should be responsibility of the children... here the children we want to modify is MenuItem, so it should have those styles.